jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/ctl/JsonConstCtl.java

```diff
@@ -25,5 +25,6 @@
     public static final String task             = "task";
     public static final String finished         = "finished";
     public static final String started          = "started";
+    public static final String success          = "success";
```

Comment:
As noted elsewhere was trying to get a `success` flag in the Task output working but couldn't get this working, left here for future reference